src/features/applet-calendar/ui/AppletCalendar.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC } from 'react';
+
+import { CalendarProvider, ExpandableCalendar } from 'react-native-calendars';
+
+import { format } from '@shared/lib/utils';
+import { Text, YStack, Box } from '@shared/ui';
+
+const AppletCalendar: FC = () => {
+  let currentDate = new Date();
+  const weekDay = format(currentDate, 'i');
+  const weekDayNumeric = +weekDay + 1;
```

Comment:
+weekDay convert string to number?  just use Number(...)

Reply:
Can we avoid data-type in names for the variables that uses in main alghorithm?
Maybe it'd be better to re-name weekDay to weekDayString ?